app/views/shared/_aside.html.slim

```diff
@@ -26,6 +26,8 @@ aside
         - if current_page?(url_for controller: 'admin/dashboard', action: 'index')
           .list-selected
     li.active
-      a[href="http://port.us.org/documentation" target="_blank"]
+      a[href="#{url_for help_index_path}"]
```

Comment:
Jus realized the whole file is not using `link_to`, so you can leave it as it is.